Armor Resistance Rebalancing

PKChrisChan opened the pull request:
Ever since armor's debut modifications in open source we've been presented with the opposite issue. Various armors being largely outclassed in many regards. Chain armor being the only useful armor.

Subsequently, with the recent nerfs to chain armor in addition to the recent additions of "Shard" gear, We've only repeated the process once again.

The goal of this change is to modify base armors usefulness in a variety of situations and furthermore enable more build/gear/inventory customizations.

This is not final, and is open for discussion and modification. We will make changes as necessary and in a unanimous agreement.

_# **The changes are as follows:**_
**Chain** is now 20% pierce resistance(up from10 percent). This will continue to be our anti arrow armor. Slash and thrust resistance have been removed. 

**Plate**: Now has a general weapon resist of 10% This means that Bludgeon, Slashing, Pierce and Thrust weapons generally do 10% less damage. The high armor value and 10% weapon resist make for good defense against these damage types.

**Scale**: This will be our anti melee armor. Slash, Thrust and Bludgeon resistances are 15%. 

**Nerudite**: Our anti-spell and anti-touch spell armor. To counterbalance, this armor now has a 10 weapon _weakness_. Additionally, it has a 50% chance to not have any armor at all versus weapons! 

Once again, none of these changes are set in stone. This is up for consideration and I will make changes based on uninamous votes. Discuss below.

PKChrisChan committed:
Resistance Rebalancing

Scale, Chain, Neru and Plate Armor all modified

Maikeljr commented:
I like these ideas, although I would look into possibly making plate weak to all magic even 5% to give it a better weakness. Maybe kill neru weakness to EQ, since weapons will be wrecking it more. I'd look into all other armors for magic resist/weakness especially ivy/orcshield/robes/other shields and just do a larger overhaul. 

Such as leather armor, can we find a niche for this? such as added spell power or resist weapon/magic resists like 5%, but 0 armor  or 200 defense or something. These micro changes will create itemization for sure, but we could go the next step and balance out the entire item selections if people want or simply balance one step at a time.

M59lost commented:
Plate needs weakness of at least 10% to acid and frost if your adding 10% across the board weapon resist.
While we are on armor, Heralds should be  changed, duke/cess should6 General sp instead of a random role of 3-7 and jonas herald should be changed to like 50 offense rating maybe 1 melee damage and like 5 kran spellpower(since mainly its pfers or similar using them anyway).

Scale looks good maybe 20% needed eventually

Neru make sense but i agree with maikel remove weakness to Earthquake. 

We added 5 sp and 25 more defense last patch maikel

Knight shields also seem unusable in most pvp situations because of 20% magic weakness, perhaps lower it  to 10% to see how it balances, 0 might be too strong as you can add 3 armor to it.

PKChrisChan committed:
Added Cold and Acid Weakness

Added a 10% and 5% acid weakness from cold and acid respectively.

PKChrisChan committed:
Knight Shield modifications

Modified from the massive 20% Faren weakness to 5% on acid, cold, and fire weakness as well 10% shock weakness. Additionally, boosted the armor value from 2 to 3.

PKChrisChan commented:
Alright, so far I've added the weakness to faren attacks on Knight Shield and Plate Armor as requested. On plate, I've added 10% cold and 5% acid and on knight shield i've removed the general magic weakness for 5% to faren and Qor's acid damage.  Also, I've given Knight shield an extra armor and 15% against all weaponcraft weapons.

TheDiggie commented:
i still argue that plate realistically is the best against arrows =p
